crates/wasmedge-sys/src/instance/module.rs

```diff
@@ -266,6 +266,19 @@ impl Instance {
         }
     }
 
+    /// Returns the host data held by the module instance.
+    pub fn host_data<T>(&self) -> Option<&mut T> {
+        let ctx = unsafe { ffi::WasmEdge_ModuleInstanceGetHostData(self.inner.0) };
+
+        match ctx.is_null() {
+            true => None,
+            false => {
+                let ctx = unsafe { &mut *(ctx as *mut T) };
+                Some(ctx)
+            }
+        }
+    }
+
```

Comment:
The Instance struct, in which host_data exists, is Send and Sync. However, host_data is obtained through an immutable borrow, which creates a mutable borrow. This means that when an Instance is cloned and sent to different threads, it's possible for multiple threads to simultaneously have &mut host_data, leading to potential data race issues.

Reply:
Accepted. Thanks a lot!